todo-app/src/App.js

```diff
@@ -1,13 +1,23 @@
 import React, { Component } from 'react';
+import todosTable from './todosTable'
+import Todo from './todo'
+import './App.css'
 
 class App extends Component {
   render() {
     return (
-      <div>
-        Hello World!
+      <div className='App'>
+        <h1>Todo List App</h1>
+        {todosTable.map((todo, index) => {
```

Comment:
Instead of using the `todosTable` directly you could have set them in the state of the `App`. For example:

```js
this.state = {
  todosTable: todosTable
};
```